from-google-translate.py

```diff
@@ -0,0 +1,4 @@
+from googletrans import Translator
+text = 'Hola Mundo!'
+translator = Translator()
+print(translator.translate(text).text)
```

Comment:
This is a nice implementation! I'll be accepting this PR for this solution after we've made a couple of changes to it.

Here's the list of changes I want you to make:

1. Can you add the version of the `googletrans` library as a comment? So that users can know what version of `googletrans` to use while running this program.
2. I need you to add an empty line at the end of this file.

Kindly implement these changes and I'll be happy to merge this PR in.